src/m365/spo/commands/listitem/listitem-add.spec.ts

```diff
@@ -204,7 +205,7 @@ describe(commands.LISTITEM_ADD, () => {
       Title: "fail adding me"
     };
 
-    await assert.rejects(command.action(logger, { options: options } as any), new CommandError("Item didn't add successfully"));
+    await assert.rejects(command.action(logger, { options: options } as any), new CommandError(`The item was not created successfully due to the following errors: ${os.EOL}- Title - failed updating`));
```

Comment:
This error message is unclear. Let's rephrase to: "Creating item has failed with the following error"